templates/service/systemd/mesos.service.erb

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Mesos <%= @name %>
+After=network.target
+Wants=network.target
+
+[Service]
+ExecStart=/usr/sbin/mesos-<%= @name %> <% if @name=='slave' -%>-containerizers=docker,mesos<% end -%>
```

Comment:
The script you refer to is not part of Mesos. Unless we'd want this module to be dependent on a 3rd party packaging (note that I'm deploying RPMs) we shouldn't rely on logic specific to it (nor to anything specific to a particular package for that matter as I think the puppet module should be agnostic to that).
if the env variables created based on that script logic make sense, perhaps we could try add it to the Mesos tree, but IMHO it offers yet another point of configuration (the 4th after cli, env, mesos conf files) and not sure it makes sense for it to exist.